Plutarch/Api/V1/Value.hs

```diff
@@ -41,6 +42,9 @@ newtype PValue (s :: S) = PValue (Term s (PMap PCurrencySymbol (PMap PTokenName
     (PlutusType, PIsData)
     via (DerivePNewtype PValue (PMap PCurrencySymbol (PMap PTokenName PInteger)))
 
+instance PEq PValue where
+  a #== b = pforgetData (pdata a) #== pforgetData (pdata b)
```

Comment:
`pdata a #== pdata b` will work here too btw

Reply:
I don't think you've fixed this

Reply:
whoops